src/vstart.sh

```diff
@@ -333,6 +334,9 @@ case $1 in
     --without-dashboard)
         with_mgr_dashboard=false
         ;;
+    -S )
+            spdk_enabled=1
```

Comment:
please the the default value at the beginning of script.

Reply:
Will set the default value to 0 at the beginning and will update script soon.